src/frontend/magnify/src/components/app/MagnifyTestingProvider/index.tsx

```diff
@@ -33,7 +33,7 @@ const locale = 'en-US';
 
```

Comment:
I think your commit message did not pass `lint-git` because commit object starts with an uppercased letter.

```diff
- 🐛(frontend) Translation of YUP messages
+ 🐛(frontend) translation of YUP messages
```